Composer/packages/client/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "start": "node scripts/start.js",
+    "start": "node --max-old-space-size=2048 scripts/start.js",
```

Comment:
Let's not commit this because it is not a problem shared across all machines. Instead, you can use `NODE_OPTIONS: --max-old-space-size=2048` env variable on your local machine if this is an issue.